app.rb

```diff
@@ -0,0 +1,174 @@
+    @people.push(student)
+    colorize_output(34, 'Person Student Created successfully')
+    display_list
+    continue
+  end
+
+  def create_teacher
+    colorize_outprint(35, 'Name: ')
+    name = gets.chomp
+    colorize_outprint(35, 'Age: ')
+    age = gets.chomp
+    colorize_outprint(35, 'Specialization: ')
+    specialization = gets.chomp
+    @people.push(Teacher.new(age: age, name: name, specialization: specialization))
+    colorize_output(34, 'Person Teacher Created successfully')
+    display_list
+    continue
+  end
+
+  def create_book
+    colorize_output(36, 'Enter Book details')
+    colorize_outprint(35, 'Title: ')
+    title = gets.chomp
+
+    colorize_outprint(35, 'Author: ')
+    author = gets.chomp
+
+    @books.push(Book.new(title, author))
+
+    colorize_output(36, 'You have successfully created your book')
+    display_list
+    continue
+  end
+
+  def list_all_books_with_numbers
+    if @books.empty?
+      colorize_output(34, 'Please enter books first!')
+      display_list
+      continue
+    else
+      @books.each_with_index do |book, index|
+        puts "#{index}) Title: #{book.title}, Author: #{book.author}"
+      end
+    end
+  end
+
+  def list_all_person_with_numbers
+    if @people.empty?
+      colorize_output(34, 'Please enter people first!!')
+      display_list
+      continue
+    else
+      @people.each_with_index do |person, index|
+        puts "#{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+  end
+
+  def create_rental()
+    colorize_output(36, 'Select a book from the following list by number')
+    list_all_books_with_numbers
+    book_option = gets.chomp.to_i
+
+    colorize_output(36, 'Select a person from the following list by number (not id)')
+    list_all_person_with_numbers
+    person_option = gets.chomp.to_i
+
+    colorize_outprint(36, 'Date: ')
+    date = gets.chomp
+    @rentals.push(Rental.new(date, @books[book_option], @people[person_option]))
+    colorize_output(36, 'Rental created successfully')
+    display_list
+    continue
+  end
+
+  def list_rentals_of_person_id()
+    print 'Person ID: '
+    id = gets.chomp.to_i
+
+    rentals = @rentals.filter { |rental| rental.person.id == id }
+    puts 'List of Rentals:'
+    rentals.each do |rental|
+      puts "Date: #{rental.date}, Book '#{rental.book.title}' by #{rental.book.author}"
+    end
+    display_list
+    continue
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def continue
+    user_option = gets.chomp
+    case user_option
+    when '1' then list_all_books
+    when '2' then list_all_people
+    when '3' then create_person
+    when '4' then create_book
+    when '5' then create_rental
+    when '6' then list_rentals_of_person_id
+    when '7'
+      colorize_output(34, 'Thank you for using this application!!')
+      exit
+    end
+  end
+  # rubocop:enable Metrics/CyclomaticComplexity
+end
```

Comment:
- [ ] Let's try not to disable Linter checks as they help us write clean and professional code. Disabling linter checks defies its purpose. Kindly fix the Linter error 👍🏽